client/galaxy/scripts/mvc/toolshed/repository-queue-view.js

```diff
@@ -27,27 +31,26 @@ var View = Backbone.View.extend({
             })
         );
         $("#center").css("overflow", "auto");
-        that.bindEvents();
+        this.bindEvents();
     },
 
     bindEvents: function() {
-        var that = this;
-        $(".install_one").on("click", function() {
-            var repository_metadata = that.loadFromQueue($(this).attr("data-repokey"));
-            that.installFromQueue(repository_metadata, $(this).attr("data-repokey"));
+        $(".install_one").on("click", ev => {
+            var repository_metadata = this.loadFromQueue($(ev.target).attr("data-repokey"));
+            this.installFromQueue(repository_metadata, $(ev.target).attr("data-repokey"));
         });
-        $(".remove_one").on("click", function() {
-            var queue_key = $(this).attr("data-repokey");
-            var repo_queue = JSON.parse(localStorage.repositories);
+        $(".remove_one").on("click", ev => {
+            var queue_key = $(ev.target).attr("data-repokey");
```

Comment:
@davebx This is an example of one of the 'gotchas' I was talking about with this/that and fat arrows. Previously 'this' would be the button, without the arrow-function. With the arrow function, we need to take the event parameter and use that to grab the target, instead of expecting 'this' from the calling context.

Reply:
Yeah, I saw that and thought to myself "oh man, `$(ev.target)` is so much clearer than `$(this)`"